app/javascript/courses/students/components/CoursesStudents__TeamsList.res

```diff
@@ -73,10 +73,17 @@ let showStudent = (team, levels, teamCoaches) => {
             <span className="leading-normal pt-px">
               {student |> TeamInfo.studentTitle |> str}
             </span>
-            <span className="font-normal ml-2 flex flex-wrap">
-              {studentTags(student)} {teamTags(team)}
+            <span className="leading-normal pt-px">
+              {switch student->TeamInfo.lastSeenAt {
+              | Some(date) =>
+                str("| Last seen: " ++ date->DateFns.formatDistanceToNowStrict(~addSuffix=true, ()))
```

Comment:
This string can go to `en.yml`. This is one of the checklist item whenever we update strings that are visible in the UI. Refer to https://developers.pupilfirst.com/docs/internationalization for the guidelines.